lib/application/README.md

```diff
@@ -0,0 +1,64 @@
+# application
+
+The application module provides an Application class that provides methods to
+queue your users' calls until the actual code to support those calls is
+present.
+
+For example, your user may expect to be able to ask your application to render
+immediately after your scout file is loaded, even though the code to perform
+the rendering has not yet loaded.
+
+Once you create an application instance, it is available via its application
+name as a property in the scout file's namespace. (By default, the scout
+file's namespace is `window.APP`, but this can be changed in your build
+configuration.) So, for example, if you create an application with a name of
+"Spotlights," then the application instance will be available at
+`window.APP.Spotlights`.
+
+The application module is typically used in conjunction with the
+[loader][1] module, [namespacer][2] module, and some configuration data:
+
+```js
+var Application = require('bv-ui-core/lib/application');
+var loader = require('bv-ui-core/lib/loader');
+var namespacer = require('bv-ui-core/lib/loader');
+var config = require('json!config.json');
+
+// Creates a namespace at window.APP and an application object at
+// window.APP.exampleApp. The config object will be available at
+// window.APP.exampleApp.config.
+namespacer.namespace('APP').registerProperty(
+  'exampleApp',
+  new Application(config)
+);
+
+// Load the core application, which will define an APP.exampleApp.onLoad method.
+loader.loadScript(config.appJS).then(APP.exampleApp.onLoad);
+```
+
+An application instance provides the following methods:
```

Comment:
This is just a nitpick: I think these docs would be much clearer if the API description was split into "users call these" and "your application calls these" sections.